로또의 최고 순위와 최저 순위.js

```diff
@@ -0,0 +1,23 @@
+function solution(lottos, win_nums) {
+    let minScore = 0;
+    let maxScore = 0;
+    const score = {
+        0: 6,
+        1: 6,
+        2: 5,
+        3: 4,
+        4: 3,
+        5: 2,
+        6: 1,
+    };
+
+    lottos.forEach((item, index) => {
+        if (item) {
+            win_nums.includes(item) ? minScore++ : 0;
+        } else {
+            maxScore++;
+        }
+    });
+    const answer = [score[minScore + maxScore], score[minScore]];
+    return answer;
+}
```

Comment:
```suggestion
function solution(lottos, win_nums) {
    const score = [6, 6, 5, 4, 3, 2, 1];
    const minScore = lottos.filter((lotto) => lotto && win_nums.includes(lotto)).length;
    const maxScore = lottos.filter((lotto) => !lotto).length;
    const answer = [score[minScore + maxScore], score[minScore]]
    return answer;
}
```
1. score 객체를 배열로 바꾸는게 좋을거 같아요
2. for문이나 foreach문을 써서 명령형 프로그래밍처럼 쓰는것보다 map, filter, reduce같은 함수형 프로그래밍 메소드를 쓰면 훨씬 더 가독성이 좋아져요. 시간복잡도가 조금 늘었지만 이렇게 쓰는게 저는 더 알아보기 쉽다고 느껴집니다. map, filter, reduce 3가지 함수를 최대한 많이 사용해보세요.